production-planning/src/main/scala/dev/atedeg/mdm/productionplanning/Actions.scala

```diff
@@ -0,0 +1,63 @@
+package dev.atedeg.mdm.productionplanning
+
+import java.time.LocalDate
+
+import cats.Monad
+import cats.data.NonEmptyList
+import cats.syntax.all.*
+
+import dev.atedeg.mdm.productionplanning.{ CheeseTypeRipeningDays, RipeningDays }
+import dev.atedeg.mdm.productionplanning.OutgoingEvent.{ OrderDelayed, ProductionPlanReady }
+import dev.atedeg.mdm.products.*
+import dev.atedeg.mdm.products.CheeseType
+import dev.atedeg.mdm.utils.*
+import dev.atedeg.mdm.utils.given
+import dev.atedeg.mdm.utils.monads.*
+
+/**
+ * Estimate how many [[Product products]] to produce that day.
+ * To do so, takes into account the [[ProductionPlan production plan]] for the same day of the previous year,
+ * the new [[Order orders]] considering the [[RipeningDays ripening time]] for each [[CheeseType cheese type]] and the
```

Comment:
Use the ubiquitous language
```suggestion
 * the new [[Order orders]] considering the [[RipeningDays ripening days]] of each [[CheeseType cheese type]] and the
```